arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java

```diff
@@ -93,26 +95,41 @@ public List<AREXMocker> queryEntryPointByRange(PagedRequestType requestType) {
     return Collections.emptyList();
   }
 
-  public List<AREXMocker> queryRecordList(ViewRecordRequestType request) {
+  public List<AREXMocker> queryRecordList(ViewRecordRequestType request, SourceProvider sourceProvider) {
```

Comment:
https://refactoringguru.cn/remove-assignments-to-parameters

Reply:
optimized, Use viewrecorddto as response model transmission